model/src/main/proto/arguments.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// The fragment arguments passed to instances of ExitProfileDialogFragment.
+message ExitProfileDialogArguments {
+  // Whether the last checked navigation drawer menu item needs to be restored(highlighted) or
+  // not after cancelling the Exit Profile Dialog.
+  bool restore_last_checked_menu_item = 1;
```

Comment:
Can this be true if admin_controls_or_nav_drawer_items is not of type last_checked_menu_item? Can you enumerate via a reply all the different behaviors we expect to be supported by this proto? I think that will better help me understand if it's structured in a way that can satisfy those requirements.